data/mods/ssb/moves.ts

```diff
@@ -265,6 +265,27 @@ export const Moves: {[k: string]: ModdedMoveData} = {
 		type: "Ice",
 	},
 
+	// Annika
+	refactor: {
+		accuracy: true,
+		basePower: 0,
+		category: "Status",
+		desc: "Raises Special Attack by 1 stage and regenerates user and foe movesets, restoring PP.",
+		shortDesc: "Regenerates user, foe movesets; +1 Sp. Atk.",
+		name: "Refactor",
+		pp: 5,
+		flags: {authentic: 1},
+		priority: -7,
+		onHit(target, source, move) {
+			for (const pokemon of this.getAllActive()) {
+				changeMoves(this, pokemon, ssbSets[pokemon.name]);
+				this.add('-message', `Annika refactored ${pokemon === target ? `herself` : `the opposing ${pokemon.name}`}!`);
```

Comment:
```suggestion
				this.add('-message', `Annika refactored ${pokemon === target ? `themself` : `the opposing ${pokemon.name}`}!`);
```

Reply:
Note: Its not just you who can use this move

Reply:
Shouldn't it also be `${target.name} refactored themselves` then?

Reply:
Im not good at words. So probably ye, ill update it